### PR TITLE
Ps/make middleware use same metric name for each service, putting service in the tags

### PR DESCRIPTION
**Pull request by Pastromhaug**
This way we make mbq.metrics middleware work the same way as atomiq/ranch/pubsub, so we can have 1 monitor for API 500 errors (and other monitors) without having to have a duplicate for every service.

### CODE REVIEW

File: mbq/metrics/__init__.py [hunk 1]
```diff
@@ -24,20 +24,22 @@
...
+_service = None
+_env = None
 _statsd = datadog.DogStatsd(
     use_default_route=True,  # assumption: code is running in a container
 )
 
 
-def init(namespace=None, constant_tags=None):
-    global _constant_tags, _initialized, _namespace
+def init(service, env, constant_tags=None):
```
Comment (Pastromhaug) [inline mbq/metrics/__init__.py]: breaking change to the interface, but we are moving to 1.0.0 anyways ;)

Comment (mattbriancon) [inline mbq/metrics/__init__.py]: `env` will be an `Environment` enum, right? So we'll need to make sure we pass the right thing when we make it a tag.

File: mbq/metrics/contrib/utils.py [hunk 2]
```diff
@@ -23,3 +25,9 @@ def get_response_metrics_tags(status_code, path, method):
         'status_code': status_code,
         'status_range': '{}xx'.format(status_code // 100),
     }
+
+
+collector = Collector(
+    namespace="mbq.metrics",
+    tags={"env": _env, "service_name": _service},
```
Comment (mattbriancon) [inline mbq/metrics/contrib/utils.py]: As written, the `env` tag will probably end up being something like `environment.production`

File: mbq/metrics/__init__.py (hunk 1 above)
Comment (Pastromhaug) [reply mbq/metrics/__init__.py]: oh I was thinking that env would be a string actually. Didn't think we wanted to make this library coupled with mbq.env. My thought was we pass in `env=ENV.long_name` when we initialize metrics. This is what we do for ranch and atomiq at least.

File: requirements-dev.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+datadog==0.24.0
+mbq.env>=0.2.0
```
Comment (mattbriancon) [inline requirements-dev.txt]: version

**Pull request merged**